src/hooks/useGetArrayState.ts

```diff
@@ -1,9 +1,6 @@
 import { useMemo } from 'react';
+import { ArrayState } from '../type/common.type.ts';
 
-interface UseGetArrayStateProps {
-  array: unknown[];
-}
-
-export const useGetArrayState = ({ array }: UseGetArrayStateProps) => {
-  return useMemo(() => (array.length === 0 ? 'EMPTY' : 'FILLED'), [array]);
+export const useGetArrayState = (array: unknown[]) => {
```

Comment:
```suggestion
export const useGetArrayState = <T>(array: T[]) => {
  return useMemo((): ArrayState => (array.length === 0 ? 'EMPTY' : 'FILLED'), [array]);
};
```
`unknown` 대신 제네릭을 사용해도 좋을 것 같네요